CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Minor: Added link to streamlink docs for easier user setup. (#4217)
 - Minor: Added setting to turn off rendering of reply context. (#4224)
 - Minor: Added setting to select which channels to log. (#4302)
+- Minor: Improved zero-width emote tooltips and rendering. (#4314)
```

Comment:
maybe better as bugfix changelogs?

`- Bugfix: Fixed an issue where animated BTTV emotes would render on top of zero-width emotes (#4314)`
`- Bugfix: Fixed an issue where it was difficult to hover a zero-width emote. (#4314)`

Reply:
Agreed